apps/admin-ui/src/realm-settings/user-profile/attribute/AddValidatorDialog.tsx

```diff
@@ -56,33 +61,39 @@ export const AddValidatorDialog = ({
         isOpen
         onClose={toggleDialog}
       >
-        <TableComposable aria-label="validators-table">
-          <Thead>
-            <Tr>
-              <Th>{t("validatorDialogColNames.colName")}</Th>
-              <Th>{t("validatorDialogColNames.colDescription")}</Th>
-            </Tr>
-          </Thead>
-          <Tbody>
-            {validators.map((validator) => (
-              <Tr
-                key={validator.name}
-                onRowClick={() => {
-                  setSelectedValidator(validator);
-                  toggleModal();
-                }}
-                isHoverable
-              >
-                <Td dataLabel={t("validatorDialogColNames.colName")}>
-                  {validator.name}
-                </Td>
-                <Td dataLabel={t("validatorDialogColNames.colDescription")}>
-                  {validator.description}
-                </Td>
+        {validators.length !== 0 ? (
+          <TableComposable aria-label="validators-table">
```

Comment:
This is not a human readable string, so let's remove it.
```suggestion
          <TableComposable>
```